### ISSUE TITLE
can orthofinder continue from clusters_OrthoFinder_I1.5.txt ?

### ISSUE THREAD

**Issue by sitaramrajaraman** [davidemms/OrthoFinder#613]
Hi David,
Unfortunately our server crashed while orthofinder was generating clusters_OrthoFinder_I1.5.txt. I managed to run mcl separately to generate this file but I have no id_pairs file. Is there a way to continue from here? It is a huge dataset and reaching this point took 2 months. So any suggestion of way of continuing from here would be great !

**Comment by davidemms** [comment]
Hi

Sorry to hear that. OrthoFinder doesn't have the functionality to do that. If most of the time was taken up for the all-v-all DIAMOND/BLAST search then you can restart from after that using "-b PREVIOUS_RESULTS_DIRECTORY".

Best wishes
David

**Comment by sitaramrajaraman** [comment]
Hi David. Sorry to reopen this but most of the time in our run was taken during the "Initital Processing of species" steps where temporary files are written to the Pickle folder. It always runs on 1 CPU. Is there a way to parallelise it?

**Comment by davidemms** [comment]
Yes, you can use the -a command line argument to parallelise this.

**Comment by sitaramrajaraman** [comment]
Hi david, i have been providing the - a option allocating 50% of the CPUs available but the parallelisation hasn't happened in that step. The pickle step runs for atleast a month in a single cpu. The threads only get activated once mcl segment starts. My installation was from conda.

**Comment by davidemms** [comment]
Hi 

There's not much I can say to this without having more information. This portion of the code has been well-used for a number of years and I haven't had anyone else suggest the parallelisation isn't working here before. As this is surprising behaviour I'd need some detailed & reliable information before starting any kind of investigation. At this stage I would suggest the basic checks such as confirming you are using the latest version, checking using 'top' or similar to examine CPU usage, testing with a smaller dataset and trying different machines. If you see parallelisation on a different computer but not on the server then that could be a first indication of any issue.

Best wishes
David
